crates/storage/libmdbx-rs/src/codec.rs

```diff
@@ -41,7 +41,7 @@ impl<'tx> TableObject for Cow<'tx, [u8]> {
 
         #[cfg(not(feature = "return-borrowed"))]
         {
-            let is_dirty = (!K::ONLY_CLEAN) &&
+            let is_dirty = (!K::IS_READ_ONLY) &&
```

Comment:
what's the difference?

Reply:
literally no difference, imo `ONLY_CLEAN` is more confusing than `IS_READ_ONLY` https://github.com/paradigmxyz/reth/blob/e1e81059fee79e9ad1931b4a30d3ce8dac8cef07/crates/storage/libmdbx-rs/src/transaction.rs#L49-L58